Automation/S3/delete_s3_buckets.py

```diff
@@ -0,0 +1,194 @@
+import boto3
+from botocore.config import Config
+from botocore.exceptions import ClientError, NoCredentialsError
+from colorama import init, Fore
+import argparse
+
+COL_SIZE = 15
+init(autoreset=True)
+
+parser = argparse.ArgumentParser(description="Deletes S3 buckets.")
+parser.add_argument('--AccessKeyId', dest='aws_access_key_id', required=True)
+parser.add_argument('--SecretAccessKey', dest='aws_secret_access_key', required=True)
+parser.add_argument('--RegionName', dest='region_name', default='us-west-2', help='Region Name (default: us-west-2)')
+
+args = parser.parse_args()
+
+def main(aws_access_key_id, aws_secret_access_key, region_name):
```

Comment:
Would be nice to have a doc comment documenting the function including inputs and outputs.

Reply:
Good point!